File: roomsplit/backend/app/utils/currency.py
def paise_to_rupees(paise: int) -> float:
    return round(paise / 100, 2)


def rupees_to_paise(rupees: float) -> int:
    return int(round(rupees * 100))


def format_inr(paise: int) -> str:
    return f"\u20b9{paise_to_rupees(paise):,.2f}"

File: roomsplit/frontend/src/utils/currency.ts
export const formatINR = (paise: number): string =>
  new Intl.NumberFormat('en-IN', { style: 'currency', currency: 'INR' }).format(paise / 100)

export const paiseToRupees = (paise: number): number => paise / 100
export const rupeesToPaise = (rupees: number): number => Math.round(rupees * 100)

export const formatRupees = (rupees: number): string =>
  new Intl.NumberFormat('en-IN', { style: 'currency', currency: 'INR' }).format(rupees)
